utils/scripts/install-el9.sh

```diff
@@ -56,8 +58,16 @@ dnf --nodocs install \
     valgrind-devel \
     which \
     yasm
+
 # No packages for the one below have been
 # identified yet. Limit build to client only for now
 #    ipmctl \
 #    libipmctl-devel \
 #    Lmod \
+#
+# ipmctl is only available on x86_64
+if [ "$arch" = x86_64 ]; then
+#    dnf --nodocs install \
+#        ipmctl \
+#        libipmctl-devel
+fi
```

Comment:
(lint) Can't have empty then clauses (use 'true' as a no-op). [SC1048]
(lint) Unexpected keyword/token. Fix any mentioned problems and try again. [SC1072]